info.json

```diff
@@ -107,36 +102,36 @@
             }
         ],
         "globalVariables": [
+            {
+                "id": 16,
+                "name": "GDPR_Demo_Email",
+                "value": "rktest.2204@outlook.com",
```

Comment:
Please remove this your email id

Reply:
This is done. The Global variable 'GDPR_Demo_Email' has been removed from SP